tests/dslash_test_utils.h

```diff
@@ -276,6 +284,36 @@ struct DslashTestWrapper {
     inv_param.verbosity = verbosity;
   }
 
+  void init_domain_decomposition(int value, int color)
+  {
+    if (value == 0) {
```

Comment:
I think we should replace `value` and `color` with some type of `enum`s as opposed to being just magic numbers to make this and other parts of the test utilities clearer. They can be `enum` that only live in the tests subdirectory, perhaps in `tests/utils/misc.h` (for lack of a better place right now), or they can leverage existing `enum` elsewhere in the code.

Reply:
I created an `enum` for `value` and `color` in `tests/utils/misc.h`